plugin/storage/integration/remote_memory_storage.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"os"
+
+	"go.uber.org/zap"
+
+	"github.com/jaegertracing/jaeger/cmd/remote-storage/app"
+	"github.com/jaegertracing/jaeger/pkg/config"
+	"github.com/jaegertracing/jaeger/pkg/healthcheck"
+	"github.com/jaegertracing/jaeger/pkg/metrics"
+	"github.com/jaegertracing/jaeger/pkg/tenancy"
+	"github.com/jaegertracing/jaeger/plugin/storage"
+	"github.com/jaegertracing/jaeger/ports"
+)
+
+type RemoteMemoryStorage struct {
+	server         *app.Server
+	storageFactory *storage.Factory
+}
+
+func StartNewRemoteMemoryStorage(logger *zap.Logger) (*RemoteMemoryStorage, error) {
+	opts := &app.Options{
+		GRPCHostPort: ports.PortToHostPort(ports.RemoteStorageGRPC),
+		Tenancy: tenancy.Options{
+			Enabled: false,
+		},
+	}
+	tm := tenancy.NewManager(&opts.Tenancy)
+	storageFactory, err := storage.NewFactory(storage.FactoryConfigFromEnvAndCLI(os.Args, os.Stderr))
+	if err != nil {
+		return nil, err
```

Comment:
wdyt about taking t.Testing arg and just calling require.NoError? This will eliminate the uncovered lines.

Reply:
Yes, this is the easiest approach. Will do